topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -182,17 +183,28 @@ We can now try to determine the library type of our data.
 
 > ### {% icon hands_on %} Hands-on: Determining the library type
 >
-> 1. Load the Ensembl gene annotation for *Drosophila melanogaster* ([`Drosophila_melanogaster.BDGP5.78.gtf`](https://zenodo.org/record/290221/files/Drosophila_melanogaster.BDGP5.78.gtf)) from [Zenodo](https://dx.doi.org/10.5281/zenodo.290221) into your current Galaxy history and rename it
+> 1. Load the Ensembl gene annotation for *Drosophila melanogaster* ([`Drosophila_melanogaster.BDGP5.78.gtf`](https://zenodo.org/record/290221/files/Drosophila_melanogaster.BDGP5.78.gtf)) from the shared data library or from [Zenodo](https://dx.doi.org/10.5281/zenodo.290221) into your current Galaxy history
+>  - Rename the dataset if necessary
+>  - Verify that the datatype is `gtf` and not `gff`. If the datatype is not `gtf`, please change it using the pencil icon.
 >
-> 2. **Select first** {% icon tool %}: Downsample both FastQ files generated by Trim Galore to 200k or 1M reads
+> 2. **Select first** {% icon tool %}: Use the tool **Select first - lines from a file**  to downsample both FastQ files generated by Trim Galore to 200k or 1M reads
 >
->    > ### {% icon question %} Question
+>    > ### {% icon question %} Questions
 >    >
->    > How many rows must be selected to conserve 200k reads?
+>    > 1. Why are we downsampling our data here?
+>    > 2. How many rows must be selected to conserve 200k reads?
 >    >
 >    > <details>
 >    > <summary>Click to view answers</summary>
->    > In a FastQ file, a read corresponds to 4 lines. So to conserve 200,000 reads, 800,000 must be selected.
+>    > <ol>
+>    > <li> We are performing a preliminary mapping for the purpose of determining the library type. We do not need the full dataset for this, and
+>    >      mapping the full dataset can take quite some time and resources, so it is better to perform this test on a small subset of the data.
+>    > </li>
+>    > <li> In a FastQ file, a read corresponds to 4 lines. So to conserve 200,000 reads, 800,000 must be selected.
+>    >      More details about the fastq format can be found <a href="https://en.wikipedia.org/wiki/FASTQ_format">here</a>.
+>    > </li>
+>    > </ol>
```

Comment:
Is the HTML formatting necessary?

Reply:
it's what was done in the other question sections and it was either this or add some `<br>` to get the second answer to start on a new line